packages/core/src/Stores/notification-store.js

```diff
@@ -180,12 +180,17 @@ export default class NotificationStore extends BaseStore {
             isAccountOfType,
             loginid,
             obj_total_balance,
+            website_status,
         } = this.root_store.client;
         const { is_p2p_visible } = this.root_store.modules.cashier.general_store;
         const { is_10k_withdrawal_limit_reached } = this.root_store.modules.cashier.withdraw;
         const { current_language, selected_contract_type } = this.root_store.common;
         const malta_account = landing_company_shortcode === 'maltainvest';
         const virtual_account = landing_company_shortcode === 'virtual';
+        const is_website_up = website_status.site_status === 'up';
+        const has_trustpilot = LocalStore.getObject('notification_messages')[loginid]?.includes(
```

Comment:
@akmal-binary you don't have to check if this notification is in localStorage here because this check is performed inside  this.addNotificationMessage when you add the notification at line 322. Or Is there any other reason for doing it twice? 🙂

Reply:
I did that first but seems like it still adds the notification into the notification bar. It looks like inside `addNotificationMessage` it just marks existing notification which will remove the popup only, but not the notification, which is not suitable in my case.